repo2docker/buildpacks/conda/environment.frozen.yml

```diff
@@ -101,31 +104,30 @@ dependencies:
   - python-json-logger=2.0.1=pyh9f0ad1d_0
   - python_abi=3.7=1_cp37m
   - pytz=2021.1=pyhd8ed1ab_0
-  - pyzmq=22.0.3=py37h499b945_0
-  - readline=8.0=he28a2e2_2
+  - pyzmq=20.0.0=py37h499b945_1
```

Comment:
A downgrade is unusual, is this expected?

Reply:
This would be due to pinning added in https://github.com/conda-forge/ipykernel-feedstock/pull/74 to block a failing pyzmq release, and should be removed if I refreeze after https://github.com/conda-forge/ipykernel-feedstock/pull/75 . Let's find out...